### PR TITLE
Added character escaping function.

### PR DESCRIPTION
**Pull request by ilyakurdyukov**
Applied to only one place in the code, should I continue further or you don't like how it's done?

### CODE REVIEW

File: mkiocccentry.c [hunk 1]
```diff
@@ -7003,3 +6992,65 @@ remind_user(char const *work_dir, char const *entry_dir, char const *tar, char c
...
+	    size += (nquot > 1 ? 2 : nquot) + (p - next) - 2;
+	}
+    va_end(va);
+    size += f - format;
+
+    cmd = malloc(size);
+    if (cmd == NULL) {
+	errp(err, func, "malloc from the cmdprintf of %ld bytes failed", size);
+	/*NOTREACHED*/
+    }
```
Comment (vog) [inline mkiocccentry.c:7025]: The indentation of the closing brace looks strange here.

**Comment by vog** [comment]
This refactoring makes sense to me, but I'm just an observer and not @lcn2.

**Comment by lcn2** [comment]
We like this pull request in genera, @vog.  Thank you.

We plan to squash & merge and then revise/refactor and apply comments / adjust as needed.

Thanks!

**Comment by lcn2** [comment]
The main issue #11 will be closed when we complete the revision and refactoring.

File: mkiocccentry.c (hunk 1 above)
Comment (ilyakurdyukov) [reply mkiocccentry.c:7025]: This is because github uses a tab with 4 spaces and the mkiocccentry source code uses a tab with 8 spaces.

**Pull request merged**